onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.cc

```diff
@@ -587,12 +585,12 @@ Status ModelBuilder::Compile(std::unique_ptr<Model>& model) {
         optype_support_status[stat_name].second++;
       }
     }
-    size_t total_ops = 0;
+    size_t fallback_ops = 0;
     std::string fallback_op_detail, normal_op_detail;
 
     for (const auto& [op, ops_status] : optype_support_status) {
       auto& [support_cnt, unspport_cnt] = ops_status;
-      total_ops += support_cnt + unspport_cnt;
+      fallback_ops += unspport_cnt;
       if (support_cnt > 0) {
```

Comment:
Are we counting every device the op is not supported on? 

e.g. 3 devices, NPU, GPU and CPU. If it's not supported on NPU and GPU are we counting this as 2 unsupported ops? 

IIUC we are calling ANeuralNetworksModel_getSupportedOperationsForDevices for non-CPU devices only, so naively I would have expected we increment the number of unsupported ops by 1 if support_cnt is 0. i.e. my assumption is that it doesn't matter how many non-CPU devices do not support the op if one of them does as NNAPI will attempt to use that device.